tests/e2e/support/api/share/share.ts

```diff
@@ -56,33 +69,28 @@ export const linkShareRoles: Readonly<{
   'Secret File Drop': 'createOnly'
 } as const
 
+const defaultRoles = {
+  viewer: { id: 'b1e2218d-eef8-4d4c-b82d-0f1a1b48f3b5', displayName: 'Can view' },
+  'space viewer': { id: 'a8d5fe5e-96e3-418d-825b-534dbdf22b99', displayName: 'Can view (space)' },
+  editor: { id: 'fb6c3e19-e378-47e5-b277-9732f9de6e21', displayName: 'Can edit without versions' },
+  'file editor': {
+    id: '2d00ce52-1fc2-4dbc-8b95-a73b73395f5a',
+    displayName: 'Can edit without versions (file)'
+  },
+  'space editor': { id: '58c63c02-1d89-4572-916a-870abc5a1b7d', displayName: 'Can edit (space)' },
+  manager: { id: '312c0871-5ef7-4b3a-85b6-0e4074c64049', displayName: 'Can Manage' },
+  uploader: { id: '1c996275-f1c9-4e71-abdf-a42f6495e960', displayName: 'Can upload' },
```

Comment:
In other places as well.
```suggestion
  'manager': { id: '312c0871-5ef7-4b3a-85b6-0e4074c64049', displayName: 'Can Manage' },
  'uploader': { id: '1c996275-f1c9-4e71-abdf-a42f6495e960', displayName: 'Can upload' },
```